sirius-pipeline/bmv2/sirius_pipeline.p4

```diff
@@ -121,6 +121,13 @@ control sirius_ingress(inout headers_t hdr,
             inbound_routing.apply();
         }
 
+        meta.dst_ip_addr = 0;
+        if (hdr.ipv6.isValid()) {
+            meta.dst_ip_addr = hdr.ipv6.dst_addr;
+        } else if (hdr.ipv4.isValid()) {
+            meta.dst_ip_addr = (bit<128>)hdr.ipv4.dst_addr;
```

Comment:
Upper bits need to be zeroed. I'm not sure if they will be zeroed by default.

Reply:
@marian-pritsak I can force set the upper bits to zero - this should be easy.

Reply:
I have found out from the P4 documentation that, in this case, the upper bits are padded to zeros.  So no changes are required.  Below is an excerpt from the P4 documentation:

<...>
8.9.1. Explicit casts
bit<W> -> bit<X>: truncates the value if W > X, and otherwise (i.e., if W <= X) pads the value with
zero bits.
<...>